css/style.css

```diff
@@ -6,3 +6,71 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+ html{
+    background-image: url(/img/images.jpeg);
+    background-size: cover;
+ }
+
+header{
+    display: flex;
+    justify-content: center;
+    justify-content: space-around;
+}
+ul{
+    display: flex;
+    gap: 2rem;
+}
+li{
+    list-style: none;
+}
+a{
+    text-decoration: none;
+}
+.logo{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+article :first-child{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```

Comment:
I would be careful with selective font styling like this - the use of different fonts is a delicate art, and you want your page to appear consistent. The way that you've gone here means that a lot of the page will be rendered with the browser default font, and then this first article will have a font that you've chosen.

It's most common to set the font at the top level, for instance:
```
body, html {
  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
}
```

This is a really good blog article on using Google Fonts on sites that might be useful here: https://www.freecodecamp.org/news/how-to-use-google-fonts-in-your-next-web-design-project-e1ad48f1adfa/